### PR TITLE
Detect go importpath on plz init

### PR DESCRIPTION
**Pull request by sagikazarmark**
This PR helps users getting started with Go projects by automatically detecting the go import path for a module.

### CODE REVIEW

**Comment by Tatskaari** [comment]
I don't know if we should be including go config by default. I'm slowly making `plz init` interactive. We should probably make it prompt the user to ask then if they would like to add language support for go.

I've done something similar for the pleasings repo.

File: src/plzinit/init.go [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+	"bufio"
 	"fmt"
-	"gopkg.in/op/go-logging.v1"
 	"io/ioutil"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
 
+	"gopkg.in/op/go-logging.v1"
```
Comment (Tatskaari) [inline src/plzinit/init.go:12]: Thanks :)

**Review by Tatskaari — APPROVED** [review]

File: src/plzinit/init.go (hunk 1 above)
Comment (sagikazarmark) [reply src/plzinit/init.go:12]: Oops, that was unintentional. :)

Comment (Tatskaari) [reply src/plzinit/init.go:12]: No no that's how we should be doing imports so it's good

Comment (sagikazarmark) [reply src/plzinit/init.go:12]: Yeah, but I don't like these changes sneaking into other changes. I prefer at least separate commits.

**Pull request merged**